docs/install.rst

```diff
@@ -1,5 +1,36 @@
 Installing JWST_REFFILES
 ========================
 
+Note that *jwst_reffiles*' main dependency, the `JWST calibration pipeline <https://github.com/spacetelescope/jwst>`_ requires Python 3.5 or above.
 
-Installation documentation goes here.
+Environment Creation
+--------------------
+
+It may be easiest to create an environment to contain *jwst_reffiles* by first creating an environment around the JWST calibration pipeline. See the installation instructions in the `JWST repo <https://github.com/spacetelescope/jwst>`_
+
+JWST_REFFILES installation
+--------------------------
+
+To download and install *jwst_reffiles*, clone from `GitHub <https://github.com/spacetelescope/jwst_reffiles>`_. Make sure you are in your desired environment, and then install using setup.py.
+
+1. git clone https://github.com/spacetelescope/jwst_reffiles.git
+2. cd jwst_reffiles
+3. python setup.py install
+
+
+Calling JWST_REFFILES
+=====================
+
+Currently, *jwst_reffiles* is designed such that it can only be called from the command line. The top-level script to be called is `mkrefs.py`. Given a :ref:`configuration file <main_cfg>`, as well as paths to collections of the necessary input files, `mkrefs.py` can be called in the following way:
+
+::
+
+  mkrefs.py -vvv --cfgfile main_config_file.cfg example_readnoise_module /path/to/dark/files/*DARK*_uncal.fits /path/to/flatfield/files/NRCN815A-LIN-53650[5678]*uncal.fits
+
+To call in this way, make sure that `mkrefs.py` is in your PYTHONPATH. Otherwise, prepend `python /path/to/mkrefs/` to the call above.
```

Comment:
even with the path to mkrefs in my command line call, it still didn't find it until I added it to my PYTHONPATH in my .bashrc file; but like I said above, I installed the old way listed (pip install .) so maybe this is fixed now with setup.py

Reply:
yeah everything was executable (except mkrefs.cfg, but I don't think that matters?) and my call specified python; I'll let you know if I notice this problem again in future runs, it could always just be a mistake on my end. But yeah, after adding it to my PYTHONPATH everything ran fine.